index.html

```diff
@@ -129,7 +129,7 @@ <h2 id="project_tagline">A framework to secure the integrity of software supply
              in-toto is maintained by <a
              href="https://github.com/in-toto/">The in-toto
              team</a>, a group of researchers and developers from <a
-             href="engineering.nyu.edu">New York University</a> and <a
+             href="http://engineering.nyu.edu">New York University</a> and <a
```

Comment:
I know that's the case now (we've reached out to them in multiple occasions), but it doesn't stop them from upgrading some day and we using https by then. I'd rather have the problem be on their side only than on our side *and* their side.